src/typescript/09-interfaces/interfaces.md

```diff
@@ -136,7 +148,7 @@ When we create empty object literals in TypeScript, they are inferred to be obje
 @sourceref ./9-4-2-type-assertion.ts
 @codepen
 
-## Objective: The use of interface for functions
+## Objective: The use of an interface for functions
```

Comment:
When there are multiple objectives on the page, go ahead and number them.

I would simplify these too. Maybe just “Define interfaces for functions” instead?